examples/tracing/ddos_detector_example.txt

```diff
@@ -0,0 +1,40 @@
+Demonstrations of ddos_detector.py, the Linux eBPF/bcc version.
+
+This tracks ip_rcv function (using kprobe) and elapsed time between received packets to detect potential DDOS attacks.
+
+
+The following steps illustrates the usage of ddos_detector :
+1 - Start ddos_detector.py :
+# ./ddos_detector.py
+DDOS detector started ... Hit Ctrl-C to end!
+TIME(s)            MESSAGE
+
+
+
+
+2 - Launch hping3 (or any other flooder) in another terminal as shown below:
+# hping3 localhost -S -A -V -p 443 -i u100
+
+
+
+3 - ddos_detector.py triggers alerts and reports a DDOS attack:
+DDOS detector started ... Hit Ctrl-C to end!
+TIME(s)            MESSAGE
+6714099756390      Attack detected
```

Comment:
I think if you can somehow convert this TIME to a human understandable time, it would be great!